docs/data/data-grid/column-groups/column-groups.md

```diff
@@ -1,18 +1,100 @@
 ---
-title: Data Grid - Column groups 🚧
+title: Data Grid - Column groups
 ---
 
-# Data grid - Column groups 🚧
+# Data Grid - Column groups
 
 <p class="description">Group your columns.</p>
 
+Grouping columns allows you to have multiple levels of columns in your header and the ability, if needed, to 'open and close' column groups to show and hide additional columns.
+
+## Define column grouping
+
+You can define the column grouping structure by providing `columnGroupingModel` prop to the data grid.
+This prop receives an array of column group.
+
+A column group is defined by at least two variables:
+
+- `groupId` a string used to identify the group
+- `children` an array containing the children of the group
+
+The children can contains two types of objects:
+
+- leafs with type `{ field: string }`, which add the column with the corresponding `field` to this group.
+- other column groups which allows you to have nested groups.
+
+:::warning
+Since this is a tree structure, a column can be associated to only one group.
```

Comment:
In a tree structure, each element can only have one parent. But it's not relevant information by itself

```suggestion
A column can only be associated with one group.
```

Reply:
For me the original sentence was here to prevent users from putting a column in two groups, thinking that it would duplicate the column (which is not forbidden by the tree structure itself).

The new sentence is fine